lib/gsheets.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Balena Ltd.
+ * Copyright 2021 Balena Ltd.
```

Comment:
```suggestion
 * Copyright 2022 Balena Ltd.
```

I think you meant 2022 :)

BTW, should we a date range in those headers instead, ie. "2020-2022"?